### PR TITLE
Added `UISearchBar` Combine publishers

### PR DESCRIPTION
**Pull request by kevinrenskers**
There seem to be a lot of changes in CombineCocoa.xcodeproj, maybe we're having an Xcode version mismatch or something? Feel free to take only UISearchBar+Combine.swift.

### CODE REVIEW

**Comment by freak4pc** [comment]
Run `make project` and commit everything created by it. Should do the trick:) 

Are you in Xcode 12 GM?

**Comment by kevinrenskers** [comment]
Xcode 12, just from the App Store.

**Comment by freak4pc** [comment]
Thanks! this needs tests

**Comment by kevinrenskers** [comment]
Tests are added, should be good to go I think.

**Comment by kevinrenskers** [comment]
I'd love to use an actual CombineCocoa release again instead of my own branch - can we merge this and cut a release? 😇 
If anything needs to be changed, please let me know!

File: Example/Podfile.lock [hunk 1]
```diff
@@ -13,4 +13,4 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: 19898029b2f3640d6c998827b076660d4767a5fd
 
-COCOAPODS: 1.9.1
+COCOAPODS: 1.9.3
```
Comment (freak4pc) [inline Example/Podfile.lock:16]: revert

File: Sources/CombineCocoa/Controls/UISearchBar+Combine.swift [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+// swiftlint:disable force_cast
+@available(iOS 13.0, *)
+public extension UISearchBar {
+    /// Combine wrapper for `UISearchBarDelegate.searchBar(_:textDidChange:)`
+    var textDidChangePublisher: AnyPublisher<String, Never> {
+        let selector = #selector(UISearchBarDelegate.searchBar(_:textDidChange:))
+        return delegateProxy
+            .interceptSelectorPublisher(selector)
+          .map { $0[1] as! String }
```
Comment (freak4pc) [inline Sources/CombineCocoa/Controls/UISearchBar+Combine.swift:21]: reindent

**Review by freak4pc — COMMENTED** [review]
two minor fixes and I'll merge it

**Review by freak4pc — APPROVED** [review]

**Pull request merged**